UnityProject/Assets/Scripts/Systems/Inventory/ItemStorage.cs

```diff
@@ -686,11 +686,25 @@ public bool ServerIsObserver(GameObject observer)
 	/// <summary>
 	/// Drops all items in all slots.
 	/// </summary>
-	public void ServerDropAll(Vector2? worldTargetVector = null)
+	public void ServerDropAll(Vector2? worldDeltaTargetVector = null)
 	{
 		foreach (var itemSlot in GetItemSlots())
 		{
-			Inventory.ServerDrop(itemSlot, worldTargetVector);
+			Inventory.ServerDrop(itemSlot, worldDeltaTargetVector);
 		}
 	}
+
+	/// <summary>
+	/// Drops all items in all slots.
+	/// </summary>
+	public void ServerDropAllAtWorld(Vector3? DropAtWorld = null)
+	{
+		Vector2? worldDeltaTargetVector = null;
+		if (DropAtWorld != null)
+		{
+			worldDeltaTargetVector =  DropAtWorld - gameObject.AssumedWorldPosServer() ;
```

Comment:
can you explain this part to me?

Reply:
is this a way to override coordiantes?

Reply:
It is because function expects the relative order to place the object To the storage, So I just made a function that turns it into relative if you Past it the world coordinate